ws_web/staff_views.py

```diff
@@ -217,6 +220,31 @@ def create(self, request, *args, **kwargs):
             )
             serializer = TagsSerializer(data=data_to_save, many=True)
             if serializer.is_valid():
+
+                # Test to make sure that the sense IDs are reasonable
+                # Especially that they are related to the gloss_with_replacement word
+                # get the pos so we can lemmatize
+                queryset = DerivedTokens.objects.filter(
```

Comment:
try/except?

Reply:
Added. Except now sends an error to Sentry either way, things get saved